### PR TITLE
Mount tmp dir for the config directory

### LINKED ISSUE #77: Failed to write global config. Error: [Errno 13] Permission denied: '/.config'.
For Rasa Open Source 2.0 in the logs, we can see the following error:

```
Failed to write global config. Error: [Errno 13] Permission denied: '/.config'. Skipping.
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-zj2d5evg because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing
```

It looks like we should add proper permissions for writing temporary files. @federicotdn @wochinge Have you seen this error before?

### PR DESCRIPTION
**Pull request by tczekajlo**
- Create a temporary directory for the Rasa global configuration
- Set `MPLCONFIGDIR` to `/tmp/.matplotlib`

Fixes #77

### CODE REVIEW

**Comment by wochinge** [comment]
ready for review?

**Comment by tczekajlo** [comment]
> ready for review?

Yes :D

File: charts/rasa-x/templates/rasa-deployments.yaml [hunk 1]
```diff
@@ -84,6 +84,8 @@ spec:
         - name: "RASA_TELEMETRY_ENABLED"
           value: "false"
         {{ end }}
+        - name: "MPLCONFIGDIR"
+          value: "/tmp/"
```
Comment (wochinge) [inline charts/rasa-x/templates/rasa-deployments.yaml:88]: I think we should use a subdirectory like 
```suggestion
          value: "/tmp/.matplotlib"
```

File: charts/rasa-x/templates/rasa-deployments.yaml [hunk 2]
```diff
@@ -136,12 +138,16 @@ spec:
         - name: "RASA_DUCKLING_HTTP_URL"
           value: "http://{{ include "duckling.host" $ }}:{{ $.Values.duckling.port }}"
         {{- include "rasa.extra.envs" $ | nindent 8 }}
-{{- if .extraVolumeMounts }}
         volumeMounts:
+        - name: config-dir
```
Comment (wochinge) [inline charts/rasa-x/templates/rasa-deployments.yaml:142]: let's add a comment that's for the global rasa config?

**Review by wochinge — APPROVED** [review]
Let's add some more descriptions to the pull request to make it clear what fixes what in this PR. 

Thanks for the fix 🚀 🙌

**Pull request merged**